scripts/rename_resources.py

```diff
@@ -26,23 +27,20 @@
 }
 
 
-def terraform_state(command: str, *args: str) -> bytes:
-    proc = subprocess.run(['terraform', 'state', command, *args],
-                          check=False,
-                          capture_output=True,
-                          shell=False)
-    sys.stderr.buffer.write(proc.stderr)
-    if proc.returncode == 0:
-        return proc.stdout
-    elif (
-        proc.returncode == 1
-        and command == 'list'
-        and b'No state file was found!' in proc.stderr
-    ):
-        log.info('No state file was found, assuming empty list of resources.')
-        return b''
-    else:
-        proc.check_returncode()
+def terraform_run(*args: str) -> str:
+    try:
+        output = terraform.run(*args).splitlines()
+    except subprocess.CalledProcessError as e:
+        if (
+            e.returncode == 1
+            and args[0:2] == ['state', 'list']
+            and 'No state file was found' in e.stderr
+        ):
+            log.info('No state file was found, assuming empty list of resources.')
+            return ''
+        else:
+            raise
+    return output
```

Comment:
Make the control flow more explicit
```suggestion
    else:
        return output
```